### DISCUSSION TITLE
STARED

### DISCUSSION THREAD

**Discussion by Abidouz** [community/community#140466] [Other Feature Feedback, Questions, & Ideas]
How do I fix the stared on my profile

**Comment by ferid333** [comment]
You mean starred ?

**Comment by ferid333** [comment]
As far as I know, Github does not offer any way for that. However, you can use a workaround. You could make a file of those repositories, or maybe a Private Repository with 1 Readme with the links to those repos. I know it is kinda like a Workaround, but that's the best way you can go about.

**Comment by ebndev** [comment]
Hi @Abidouz, thanks for being a part of the GitHub Community!

You are more likely to get a useful response if you share more information about your issue. Giving a few more details might help someone give you a nudge in the right direction. 😄

**Comment by venujaranasinghe** [comment]
To remove the "starred" repositories from your GitHub profile, you can follow these steps:

Go to Your GitHub Profile:

Navigate to [github.com](https://github.com/) and log in if you’re not already.
Access Your Starred Repositories:

On your profile page, look for the "Stars" tab under your username. This tab shows all the repositories you've starred.
Unstar Repositories:

Click on the "Stars" tab to see the list of repositories you've starred.
To remove a star, click the star icon next to each repository you want to unstar. The icon will change to an outline, indicating that the repository is no longer starred.
After you've unstarred the repositories, they will no longer appear in the starred section of your GitHub profile. If you want to clean up your profile further, you can also manage other sections like followers, following, and pinned repositories in your profile settings.
